addon/utils.js

```diff
@@ -23,45 +11,18 @@ async function resultOfEval(codeString) {
 }
 
 /**
- * Return an backward iterator over an Array.
+ * Deletes all children of the specified element.
  */
-function *reversed(array) {
```

Comment:
Let's not delete this; it's handy in some rulesets. (Some of the utils code is in fact dead code, meant for use by clients. Though, of course, we shoudl have tests for it all in an ideal world.)

Reply:
ah!